app/controllers/api/v1/users_controller.rb

```diff
@@ -50,6 +92,12 @@ def by_location
 
       private
 
+      def set_social_user_and_redirect
+        user_and_redirect = User.from_social(params[:user])
```

Comment:
Based on the comment about, you can delete this method.

Reply:
done.